### DISCUSSION TITLE
Cancel a comment when you are adjusting stock fail

### DISCUSSION THREAD

**Discussion by ferventurart** [Blair2004/NexoPOS#272] [Gastro 4.x Ideas & Issues] [answered]
![image](https://user-images.githubusercontent.com/16241095/130142953-881a4dd6-c232-4de2-a866-66706e5c98ee.png)

When you try to add a comment in a adjustment stock module, and you click cancel, the app is crashing.

**Comment by Blair2004** [comment, marked as answer]
The issue has been fixed on recent version
